test/over_react/component_declaration/builder_integration_tests/component2/constant_required_accessor_integration_test.dart

```diff
@@ -11,127 +11,182 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import 'dart:html';
+import 'dart:js';
 
 import 'package:over_react/over_react.dart';
-import 'package:over_react/react_dom.dart' as react_dom;
+import 'package:react/react_client/react_interop.dart' show PropTypes;
 import 'package:test/test.dart';
 
 import '../../../../test_util/test_util.dart';
 
 part 'constant_required_accessor_integration_test.over_react.g.dart';
 
 void main() {
-  group('(Component2) properly identifies required props by', () {
+  group('(Component2) propTypes properly identifies required props by', () {
+    List<String> consoleErrors;
+    JsFunction originalConsoleError;
+
+    setUp(() {
+      consoleErrors = [];
+      // PropTypes by default will only throw a specific error one time per Component Class.
+      // This resets the cache after each test so it throws again.
+      // See: https://www.npmjs.com/package/prop-types#proptypesresetwarningcache
+      PropTypes.resetWarningCache();
+
+      originalConsoleError = context['console']['error'];
+      context['console']['error'] = new JsFunction.withThis((self, message) {
+        consoleErrors.add(message);
+        originalConsoleError.apply([message], thisArg: self);
+      });
+    });
+
+    tearDown(() {
+      context['console']['error'] = originalConsoleError;
+    });
```

Comment:
#nit missing newline after

Reply:
i put the missing newline over here: https://github.com/Workiva/over_react/pull/317#discussion_r310211798